src/code-templates/functional-api-client/FunctionalApiClient/ArrowFunction.ts

```diff
@@ -121,14 +121,35 @@ const methodTypeParameters = (factory: TsGenerator.Factory.Type, { convertedPara
  */
 export const create = (factory: TsGenerator.Factory.Type, params: CodeGenerator.Params, option: Option): ts.ArrowFunction => {
   const { convertedParams } = params;
-  const typeParameters: ts.TypeParameterDeclaration[] = methodTypeParameters(factory, params);
+  const typeParameters: ts.TypeParameterDeclaration[] = [];
+  typeParameters.push(
+    factory.TypeParameterDeclaration.create({
+      name: "RequestOption",
+    }),
+  );
+  typeParameters.push(...methodTypeParameters(factory, params));
```

Comment:
each functions are now accept the `apiClient` argument and `RequestOption` type argument.